src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -49,7 +49,7 @@ public static List<PartyDetails> buildHearingPartiesDetails(HearingWrapper wrapp
         }
 
         if (isYes(caseData.getJointParty().getHasJointParty())) {
-            partiesDetails.add(createJointPartyDetails(caseData));
+            partiesDetails.addAll(buildHearingPartiesPartyDetails(caseData.getJointParty(),appeal.getHearingOptions(),appellant.getId()));
```

Comment:
Just a tidiness comment for future considerations, all on one line makes it hard to read.  Prefer to layout easier to read such as:
partiesDetails.addAll(
  buildHearingPartiesPartyDetails(
       caseData.getJointParty(),
       appeal.getHearingOptions(),
       appellant.getId()));

Reply:
(it looks better with the right indentation!!!)